packages/insomnia/src/ui/routes/git-actions.tsx

```
@@ -467,12 +466,12 @@ export const cloneGitRepoAction: ActionFunction = async ({
   });
   repoSettingsPatch.needsFullClone = true;
 
-  const fsClient = MemClient.createClient();
+  const inMemoryFsClient = MemClient.createClient();
 
   const providerName = getOauth2FormatName(repoSettingsPatch.credentials);
   try {
     await shallowClone({
-      fsClient,
+      fsClient: inMemoryFsClient,
       gitRepository: repoSettingsPatch as GitRepository,
     });
   } catch (e) {
```

If we still need `shallowClone` here?

---

I think we do in case there are many workspaces or if the workspace we try to clone already exists and we want to throw an error.